### ISSUE TITLE
Build failure with GHC 7.10.3

### ISSUE THREAD

**Issue by RyanGlScott** [andrewthad/quickcheck-classes#51]
From [`primitive`'s Travis CI config](https://travis-ci.org/haskell/primitive/jobs/429858818#L1267):

```
[13 of 31] Compiling Test.QuickCheck.Classes.MonadPlus ( src/Test/QuickCheck/Classes/MonadPlus.hs, /tmp/dist-test.ghHs/dist-newstyle/build/x86_64-linux/ghc-7.10.3/quickcheck-classes-0.4.14.2/build/Test/QuickCheck/Classes/MonadPlus.o )

src/Test/QuickCheck/Classes/MonadPlus.hs:63:42:
    Not in scope: type constructor or class ‘LinearEquationM’
    Perhaps you meant one of these:
      ‘LinearEquation’ (imported from Test.QuickCheck.Classes.Common),
      ‘LinearEquationTwo’ (imported from Test.QuickCheck.Classes.Common)
```

**Comment by chessai** [comment]
I believe that this is fixed in the newest version of quickcheck-classes on
hackage. I ran into this issue when working on a not-yet-merged branch for
primitive, and I believe it was fixed by changing the version of
quickcheck-classes in test/primitive-test.cabal

**Comment by RyanGlScott** [comment]
I still experience this error locally as of `quickcheck-classes-0.4.14.2`.

**Comment by chessai** [comment]
Oh, I should have looked at the output more carefully, I apologise.
Disregard what I said. Let me look into this. I'm in a doctor's waiting
room right now. Maybe @andrewthad can get it done faster, but it'll give me
something to do.

**Comment by chessai** [comment]
I think it's because the CPP for exporting LinearEquationM from
Test.QuickCheck.Classes.Common is wrong. It says to export it if the
version of base is 4.9.0 or more, but it should be 4.8.

**Comment by andrewthad** [comment]
I'm looking at this now.

**Comment by andrewthad** [comment]
This should be fixed by ade4948ef778488074763e9376b27dec3d5bbe97. I have released this to hackage as quickcheck-classes-0.4.14.3.

**Comment by RyanGlScott** [comment]
That works for me. Thanks for the quick fix!
